Update devquests.md (#110)
RetroAchievements/docs: docs/developer-docs/devquests.md

4. Homebrew that is essentially the same as another you've done will not count (i.e. multiple Flappy Bird clones can only garner credit once)
5. One set per large homebrew collection maximum (i.e. One Net Yaroze 2014 Collection set allowed)

### 22 [This Belonged in a Museum!](https://retroachievements.org/game/25678)

![DQ022](https://retroachievements.org/Images/102134.png)

**Type**: Revision, Maintenance 
**Goal**: Obtain 4 points by dusting off older sets, bringing them up to current RA standards  
**Repeatable**: No

> 1. The Quest is completed upon obtaining 4 points towards it. Full Set Dust-Off = 1 Point, Misc Task = 0,25pt, at least one set dust-off required for the badge.

Reviewer: Probably should spell it out as 0.25 points (American-centric, I know), or as "a quarter point." Or use the 1/4 Unicode character.

Author: Per internal DQ Team talk, updating the point framework to require 12 Total, meaning the dust-off is now worth 4pts and the misc 1pt each.

> 1. The Quest is completed upon obtaining 12 points towards it. Full Set Dust-Off = 4 Points, Misc Task = 1 Point, at least one set dust-off required for the badge.

Reviewer: Consider making the point descriptions sub-bullets for more visibility.

Author: Done

> 1. The Quest is completed upon obtaining 12 points towards it. Full Set Dust-Off = 4 Points, Misc Task = 1 Point, at least one set dust-off required for the badge.

Reviewer: ```suggestion
1. The quest is completed upon obtaining 12 points towards its goals. Full Set Dust-Off = 4 Points, Miscellaneous Task = 1 Point, at least one set dust-off required for the badge.
```

> 2. Full Set Dust-Off: Clean up old code notes, update achievement logics for modern best practices (removing unnecessary Hits and Resets to use Delta checks instead for example), new cleaner badges, updated descriptions and achievements titles, new dynamic RP.

Reviewer: ```suggestion
2. Full Set Dust-Off: Clean up old code notes, update achievement logic for modern best practices (removing unnecessary Hits and Resets to use Delta checks instead, for example), update with cleaner badges, update descriptions and achievement titles, new dynamic RP.
```
Just some subject/verb tense agreement updates

> 3. Misc Tasks: Update a set in order to remove it from the [No Dynamic RP](https://retroachievements.org/game/3083), [Player-Input RP](https://retroachievements.org/game/28506) or [Single Save Support](https://retroachievements.org/game/28449) Hubs. Doesn't stack with a Full Set Clearing.

Reviewer: Consider putting the three hubs as sub-bullets for readability

Author: Done

4. No Collaboration allowed.

> 5. This Quest do not stack with other revision DevQuest. If a set would also be eligible to DQ2 or DQ13 they will take priority.

Reviewer: ```suggestion
5. This quest does not stack with other revision DevQuests. If a set would also be eligible to DQ2 or DQ13 those quests will take priority.
```

6. Sets are subject to DevQuest team approval, along with a small plan on what the dust-off will be about. Sets already in the approved DQ22 list do not require approval, only notification.

> 7. Badges update must go through the Icon-Gauntlet revision process.

Reviewer: ```suggestion
7. Badge updates must go through the Icon-Gauntlet revision process.
```

8. Revision rules apply as usual, meaning if you only clean-up the logic and don't change the achievements scope you don't need one.

> 9. Devs own sets made prior to 2024 are eligibles for half the points on a Full Dust-Off. Similarly to sets not pre-approved, they will need to be sent to the DevQuest team for approval.

Reviewer: ```suggestion
9. Devs' own sets made prior to 2024 are eligible for half the points on a Full Dust-Off. As with sets that are not pre-approved, they will need to be sent to the DevQuest team for approval.
```
